File: fix.py
import sys

with open('src/services/paymentEngine.js', 'r', encoding='utf-8') as f:
    code = f.read()

code = code.replace("id: ", "id: ev_")
code = code.replace("", "")
code = code.replace("note: Reversal of : ,", "note: Reversal of : ,")
code = code.replace("note: Reversal of : ,", "note: Reversal of : ,")

with open('src/services/paymentEngine.js', 'w', encoding='utf-8') as f:
    f.write(code)

print("Python replace done")

File: fix.js
const fs = require('fs');
let code = fs.readFileSync('src/services/paymentEngine.js', 'utf8');
code = code.replace(/id: \\$\{ev__\}\$\{paymentId\}_\\$\{Date\.now\(\)\}\/, 'id: \ev_\$\{paymentId\}_\$\{Date.now()\}\');
fs.writeFileSync('src/services/paymentEngine.js', code);